docs/ORGANIZATION.md

```diff
@@ -1,21 +1,108 @@
 # Organization of Documentation for napari
 
-The organization of documentation for the napari project is currently split into three repositories:
-the [`napari/napari`] repo which houses the source code for napari, the [`napari/docs`] repo which contains legacy and version-specific API documentation and guides, and the [`napari/napari.github.io`] repo that makes up the main [napari.org] site, comprising of tutorials and copied-over files from the [`napari/napari`] repo.
+The organization of documentation for the napari project is currently split into
+three repositories: the [`napari/napari`] repo, which houses the source code for
+napari, the [`napari/docs`] repo which contains legacy and version-specific API
+documentation and guides, and the [`napari/napari.github.io`] repo that makes up
+the main [napari.org] site, comprising of tutorials and copied-over files from
+the [`napari/napari`] repo.
```

Comment:
It might be nice to split the three repository description into bullet points to make this easier to reference?